src/modem/at.c

```diff
@@ -253,7 +253,20 @@ static void at_task_run_bytes_read(struct at_info *ati, char *msg)
          * Our rx state will dictate how we process this message beacuse that
          * allows us to know previous messages and what message type to expect.
          */
-        switch (ati->rx_state) {
+        //enum at_rx_state state = ati->rx_state;
```

Comment:
And hell you just saved my butt.  That was some code that I left in there for testing that should have been removed.  Thanks :)